WindowsAzureMessaging/WindowsAzureMessaging/Config/Global.xcconfig

```diff
@@ -0,0 +1,173 @@
+#include "./Version.xcconfig"
+
+// :Mark: Architectures
+MS_WATCH_ARCHS = armv7k arm64_32
+MS_SIM_ARCHS = x86_64 i386
+SDKROOT = iphoneos
+
+// :Mark: Deployment
+IPHONEOS_DEPLOYMENT_TARGET = 10.0
+TVOS_DEPLOYMENT_TARGET = 11.0
+MACOSX_DEPLOYMENT_TARGET = 10.9
+COPY_PHASE_STRIP = NO
+
+// :Mark: Packaging
+PRODUCT_NAME = $(PROJECT_NAME)
+
+// :Mark: Language configurations
+CLANG_ENABLE_MODULES = NO
+CLANG_ENABLE_OBJC_ARC = YES
+CLANG_CXX_LIBRARY = libc++
+CLANG_CXX_LANGUAGE_STANDARD = gnu++14
+GCC_C_LANGUAGE_STANDARD = gnu99
+
+// :Mark: Search paths
+ALWAYS_SEARCH_USER_PATHS = NO
+//USER_HEADER_SEARCH_PATHS = "$(SRCROOT)/../AppCenter/AppCenter"/**
+
+// :Mark: Code signing
+CODE_SIGN_IDENTITY =
+
+// :Mark: Linking
+DEAD_CODE_STRIPPING = NO
+
+// :Mark: Code generation
+GCC_NO_COMMON_BLOCKS = YES
+
+// :Mark: Preprocessing
+ENABLE_STRICT_OBJC_MSGSEND = YES
+
+// This flag adds -fembed-bitcode-marker in the release configuration
+// but we require the full bitcode flag -fembed-bitcode in this case.
+ENABLE_BITCODE = NO;
```

Comment:
This is wrong settings that break Carthage on tvOS, it's already fixed in SDK, pick up the changes